.github/workflows/CI.yml

```diff
@@ -18,6 +18,7 @@ jobs:
         run: |
           source /opt/ros/galactic/setup.bash
           sudo apt-get update
+          sudo apt-get install -y protobuf-compiler
```

Comment:
This can be installed by adding `protobuf-dev` as a dependency to the package that needs it. (This is the rosdep key that includes the `protobuf-compiler` apt package).